src/ZXing.Net.Maui/ZXing.Net.MAUI.Controls/Controls/CameraView.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using System;
 using Microsoft.Maui.Controls;
 using Microsoft.Maui.Graphics;
-using System;
```

Comment:
Should we fix this in the commit that added ZXing in?

Reply:
Again. Why?

Reply:
Either:
1- you fix it where you added the file
2- you fix it in a separate commit
fixing it in a non-related commit is not the way

Reply:
Extracted to a separate commit.